### PR TITLE
[db] change user model

### PR DESCRIPTION
**Pull request by Infinidrix**

### CODE REVIEW

File: Hack/Backend/controllers/LocationUserController.js [hunk 1]
```diff
@@ -28,18 +28,20 @@ exports.post_location_user = async (req, res) => {
...
     });
     try {
       let result1 = await axios.get(`https://api.mapbox.com/geocoding/v5/mapbox.places/${location.longitude},${location.latitude}.json?types=poi&access_token=pk.eyJ1IjoiZmVyb3g5OCIsImEiOiJjazg0czE2ZWIwNHhrM2VtY3Y0a2JkNjI3In0.zrm7UtCEPg2mX8JCiixE4g`)
         .then(response => {
           if (response.data) {
             if (response.data.features && response.data.features.length > 0) {
-              location.longitude = response.data.features[0].center[0];
-              location.latitude = response.data.features[0].center[1];
-              location.place_name = response.data.features[0].text;
+              location.location.longitude = response.data.features[0].center[0];
```
Comment (Khalid-Sultan) [inline Hack/Backend/controllers/LocationUserController.js:42]: location.location.coordinates[0] = response.data.features[0].center[0];
              location.location.coordinates[1] = response.data.features[0].center[1];
I think it should be this way right

File: Hack/Backend/controllers/LocationController.js [hunk 2]
```diff
@@ -135,27 +135,28 @@ exports.post_location = async (req, res) => {
...
     });
     try {
       const result = await axios.get(`https://api.mapbox.com/geocoding/v5/mapbox.places/${location.longitude},${location.latitude}.json?types=poi&access_token=pk.eyJ1IjoiZmVyb3g5OCIsImEiOiJjazg0czE2ZWIwNHhrM2VtY3Y0a2JkNjI3In0.zrm7UtCEPg2mX8JCiixE4g`)
         .then(response => {
           if (response.data) {
             if (response.data.features && response.data.features.length>0) {
-              location.longitude = response.data.features[0].center[0];
-              location.latitude = response.data.features[0].center[1];
-              location.place_name = response.data.features[0].text;
+              location.location.longitude = response.data.features[0].center[0];
```
Comment (Khalid-Sultan) [inline Hack/Backend/controllers/LocationController.js:157]: location.location.coordinates[0] = response.data.features[0].center[0];
              location.location.coordinates[1] = response.data.features[0].center[1];
I am not sure if we can modify longitude and latitude that way

Comment (Infinidrix) [reply Hack/Backend/controllers/LocationController.js:157]: Yes yes, you're right

File: Hack/Backend/controllers/LocationController.js [hunk 3]
```diff
@@ -135,27 +135,28 @@ exports.post_location = async (req, res) => {
...
       const result = await axios.get(`https://api.mapbox.com/geocoding/v5/mapbox.places/${location.longitude},${location.latitude}.json?types=poi&access_token=pk.eyJ1IjoiZmVyb3g5OCIsImEiOiJjazg0czE2ZWIwNHhrM2VtY3Y0a2JkNjI3In0.zrm7UtCEPg2mX8JCiixE4g`)
         .then(response => {
           if (response.data) {
             if (response.data.features && response.data.features.length>0) {
-              location.longitude = response.data.features[0].center[0];
-              location.latitude = response.data.features[0].center[1];
-              location.place_name = response.data.features[0].text;
+              location.location.coordinates[0] = response.data.features[0].center[0];
+              location.location.coordinates[1] = response.data.features[0].center[1];
+              location.location.place_name = response.data.features[0].text;
```
Comment (Khalid-Sultan) [inline Hack/Backend/controllers/LocationController.js:159]: location.place_name and not location.location.place_name if I'm not mistaken

File: Hack/Backend/controllers/LocationUserController.js [hunk 4]
```diff
@@ -28,18 +27,20 @@ exports.post_location_user = async (req, res) => {
...
       let result1 = await axios.get(`https://api.mapbox.com/geocoding/v5/mapbox.places/${location.longitude},${location.latitude}.json?types=poi&access_token=pk.eyJ1IjoiZmVyb3g5OCIsImEiOiJjazg0czE2ZWIwNHhrM2VtY3Y0a2JkNjI3In0.zrm7UtCEPg2mX8JCiixE4g`)
         .then(response => {
           if (response.data) {
             if (response.data.features && response.data.features.length > 0) {
-              location.longitude = response.data.features[0].center[0];
-              location.latitude = response.data.features[0].center[1];
-              location.place_name = response.data.features[0].text;
+              location.location.coordinates[0] = response.data.features[0].center[0];
+              location.location.coordinates[1]  = response.data.features[0].center[1];
+              location.location.place_name = response.data.features[0].text;
```
Comment (Khalid-Sultan) [inline Hack/Backend/controllers/LocationUserController.js:43]: location.place_name and not location.location.place_name if I'm not mistaken

File: Hack/Backend/controllers/LocationController.js (hunk 3 above)
Comment (Infinidrix) [reply Hack/Backend/controllers/LocationController.js:159]: Yes you're right

**Pull request merged**